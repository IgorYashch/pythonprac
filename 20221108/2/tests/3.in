("10", "10"), (4, 5), (1, 2)
1,2,3,4,5,6
(-1, -1), (1, 1), (2, 2)
lfdsaksdasfdaj
(0.3, 0.7), (1, 1), (100, 0)