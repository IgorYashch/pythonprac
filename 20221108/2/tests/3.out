Not a triangle
Invalid input
Not a triangle
Invalid input
15.2
